src/Components/Main.tsx

```diff
@@ -90,6 +102,38 @@ export const Main = (props: IProps) => {
     if (loadingXLSX) onShare();
   }, [loadingXLSX]);
 
+  React.useEffect(() => {
+    let _global = 0;
```

Comment:
can we add a new prop to disable this feature?

Reply:
Done